cypress.config.js

```diff
@@ -0,0 +1,12 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+  },
+  env: {
+    API_URL: 'http://localhost:3001',
+  },
+  fixturesFolder: false,
+});
```

Comment:
```suggestion
});

```

Recomenda-se deixar uma linha em branco ao final de cada arquivo.

Dessa forma, evita-se que ferramentas como o GitHub exibam o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção que deve ser seguida, indepedente da extensão do arquivo.